tests/system/action/meeting_user/test_update.py

```diff
@@ -0,0 +1,25 @@
+from tests.system.action.base import BaseActionTestCase
+
+
+class MeetingUserUpdate(BaseActionTestCase):
+    def test_update(self) -> None:
+        self.set_models(
+            {
+                "meeting/10": {
+                    "is_active_in_organization_id": 1,
+                    "meeting_user_ids": [5],
+                },
+                "meeting_user/5": {"user_id": 1, "meeting_id": 10},
+            }
+        )
+        test_dict = {
+            "id": 5,
+            "comment": "test bla",
+            "number": "XII",
+            "structure_level": "A",
+            "about_me": "A very long description.",
```

Comment:
Needs discussion:
Write one more test to change the about_me-field without any right, but being member of the meeting as replacement for the field in user.update_self. You also have to change the permission check.